### DISCUSSION TITLE
That video player is very GOOD player for 4k!

### DISCUSSION THREAD

**Discussion by n0bzeR** [zaps166/QMPlay2#686] [General]
i have old pc:
gtx 570
i3 550 x64 bit
monitor: 1600x900 (900p)
AND THIS PLAYER CAN PLAY 4K VIDEOS IN 60 FPS !!! 
i dont know , WHY IT POSSIBLE, on my hardware, but it COOL!
MPV player much , much lagging, when i try run 4k video, 60 fps :D
qmplay2 not lagging, what that magick ? o.o
for 4k videos, QMPLAY2 - amazing :)))

**Comment by zaps166** [comment]
Good to hear it's working good and fast :slightly_smiling_face: Thanks!
